src/core/api/index.js

```diff
@@ -305,3 +306,15 @@ export const allPages = async (
   // If we get this far the callback may not be advancing pages correctly.
   throw new Error(`Fetched too many pages (the limit is ${pageLimit})`);
 };
+
+export const validateLocalizedString = (localizedString: LocalizedString) => {
```

Comment:
According to the API docs, if we're sending the `lang` parameter along with the request (which we should be), we shouldn't be getting an object back, just a string in the correct locale. (http://addons-server.readthedocs.io/en/latest/topics/api/overview.html#translated-fields)

Is this because we are sending multiple translated fields?

Reply:
Ah, just read below, looks like yes. Okay, cool!

Reply:
Technically though, we're allowed to send a single string to the API if we supply the locale... and I think we shouldn't be supplying the locale (which I'm pretty sure `callApi` does automatically) if we're sending an object.